recipes/el-secretario-mu4e

```diff
@@ -0,0 +1,4 @@
+(el-secretario-mu4e
+ :fetcher git
+ :url "https://git.sr.ht/~zetagon/el-secretario"
+ :files ("el-secretario-mu4e.el"))
```

Comment:
I still need to get the manual mu4e install working on my checks, but I believe the following warnings are correct --

`el-secretario-mu4e.el` with byte-compile using Emacs 27.1:
```
el-secretario-mu4e.el:60:11:Warning: assignment to free variable
    `el-secretario-mu4e--activate-backwards'
el-secretario-mu4e.el:82:1:Warning: Unused lexical argument `obj'
el-secretario-mu4e.el:87:1:Warning: Unused lexical argument `obj'
In end of data:
el-secretario-mu4e.el:121:1:Warning: the following functions are not known to
    be defined: el-secretario-activate-keymap,
    el-secretario--previous-source,
    el-secretario--next-source
```